Logger/FileLogger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string fPath;
+        StreamWriter sWriter;
+
+        public FileLogger(string pth)
+        {
+            fPath = pth;
+
+            if(File.Exists(fPath))
+            {
+
+            }
+            else
+            {
+                File.Create(fPath);
+            }
+        }
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            DateTime dateTime = DateTime.Now;
+            string dTime = dateTime.ToString();
+            string lLevel = logLevel.ToString();
+
+            string logInput = $"{dTime} {className} {lLevel} {message}";
```

Comment:
Variable inside of string interpolation will automatically get converted to string for you. You do not need to explicitly call `.ToString()` on them.
```suggestion
            string logInput = $"{DateTime.Now} {className} {logLevel} {message}";
```